.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,67 +1,72 @@
+_[Enter an explanation of what the pull request does and why.]_
 
-## Additions
 
--
-
-## Removals
-
--
+---
 
-## Changes
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
 
--
 
-## Testing
+## How to test this PR
 
 1.
 
+
 ## Screenshots
 
 
-## Notes
+## Notes and todos
 
 -
 
-## Todos
-
--
 
 ## Checklist
 
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
+
 - [ ] PR has an informative and human-readable title
+  - PR titles are used to generate the change log in [releases](../../releases),
+    so good ones make that easier to scan.
+  - Consider prefixing with the feature/section you're addressing,
+    e.g., "Mega Menu: fix layout bug" or "Paying for College: Update content on Repay tool"
 - [ ] Changes are limited to a single goal (no scope creep)
-- [ ] Code can be automatically merged (no conflicts)
 - [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
-- [ ] Passes all existing automated tests
-- [ ] Any _change_ in functionality is tested
-- [ ] New functions are documented (with a description, list of inputs, and expected output)
-- [ ] Placeholder code is flagged / future todos are captured in comments
-- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
+- [ ] Future todos are captured in comments and/or tickets
 - [ ] Project documentation has been updated
-- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:
+  - Potential places for documentation:
+    - [This repo’s docs](https://cfpb.github.io/cfgov-refresh/) (edit the files in the `/docs` folder) –
+      for basic, close-to-the-code documentation on working with this repo
+    - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
+    - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
 
-## Testing checklist
+### Front-end testing
 
-### Browsers
+_[When new or significantly modified front-end functionality is present,
+  the following things should be tested. Feel free to delete this section
+  if not applicable to this PR.]_
 
-- [ ] Chrome on desktop
+#### Browser testing
+
+Visually tested in the following supported browsers:
 - [ ] Firefox
-- [ ] Safari on macOS
-- [ ] Edge
-- [ ] Internet Explorer 9, 10, and 11
+- [ ] Chrome
+- [ ] Safari
+- [ ] Edge 18
+- [ ] Internet Explorer 11 and 8 (via emulation in 11's dev tools)
 - [ ] Safari on iOS
 - [ ] Chrome on Android
 
-### Accessibility
+[Further guidance on browser support](https://github.com/cfpb/development/blob/master/guides/browser-support.md)
+
+#### Accessibility
 
-- [ ] Keyboard friendly
+- [ ] Keyboard friendly (navigable with tab, space, enter, arrow keys, etc.)
 - [ ] Screen reader friendly
+- [ ] Does not introduce new errors or warnings in
+      [WAVE](https://wave.webaim.org/extension/)
```

Comment:
This just seemed like it might be a nice addition. Thoughts?

Reply:
Do we do WAVE testing at the PR level or is it on more of a project timeline? As much as I think that we should test in WAVE it feels unlikely that we are doing this on every UI PR. Maybe we should, but I'm reluctant to introduce it as a standard/expectation by adding it to the PR template. That said, I'm very open to discussion on this!

Reply:
We don't currently have any regular process for doing it other than during accessibility audits.

My thinking here is that it would just sort of be another quick eyeball test when opening a PR: If you messed with HTML or CSS, open the WAVE toolbar and look at the part of the page where your changes occurred. If you see anything related to the work you did, fix it.

I would prefer to make it into an automated check of some kind, but I haven't really researched how possible that is.